classes/DwcArchiverAssociation.php

```diff
@@ -98,17 +98,29 @@ private function trimBySchemaType($dataArr){
 
 	// @TODO decide if setDynamicFields is needed
 
-	private function setSqlBase(){
+	public function setSqlBase($getInverse = false){
 		if($this->fieldArr){
 			$sqlFrag = '';
-			foreach($this->fieldArr['fields'] as $colName){
-				if($colName) $sqlFrag .= ', ' . $colName;
+			if(!$getInverse){
+				foreach($this->fieldArr['fields'] as $colName){
+					if($colName) $sqlFrag .= ', ' . $colName;
+				}
+				// $this->sqlBase = 'SELECT ' . trim($sqlFrag, ', ') . ' FROM omoccurassociations ';
+				$this->sqlBase = 'SELECT DISTINCT ' . trim($sqlFrag, ', ') . ' FROM omoccurrences o INNER JOIN omoccurassociations oa ON o.occid = oa.occidAssociate ';
```

Comment:
I think that this should have been o.occid = oa.occid. Looks like changing it in 1924-export-associations-as-resource-relationship fixed some bugs I was seeing.